pyriemann/preprocessing.py

```diff
@@ -200,3 +202,63 @@ def inverse_transform(self, X):
         """
         Xiw = self.inv_filters_.T @ X @ self.inv_filters_
         return Xiw
+
+
+def class_distinctiveness(X, y, nume_denomi=False):
+    """Measure class separability between classes on a manifold.
+
```

Comment:
Can you write the formula for class distinctiveness value?

Reply:
Actually, I’m not super comfortable with the current formula of  class distinctiveness. 
IMO, I think all distances should be at a power of 2.
We can _solve_ this uncertainty, adding a parameter `p` being the exponent of power.

Reply:
This is a quick update, but when I was writing the formula for the documentation, I realized that my implementation of the second commit did not correctly reflect the equation of multi-class distinctiveness (Equation 7 in the original paper). I am sorry for that. I will modify it in the next commit.

Reply:
Actually, I am not sure if I fully understood your suggestion of using parameter p. That would be a different equation than the original paper's one, wouldn't it? 
[https://iopscience.iop.org/article/10.1088/1741-2552/aac577/meta?casa_token=bZ_Sz-fcfXIAAAAA:_lr-H99Z4_Y-GrbxMgL6ELEOvx03b8I4Xa7gedXTciHcUdvz5TzQhGCZ1WGcKqNOOpWWRmbVKXx5](https://github.com/pyRiemann/pyRiemann/pull/url)

Reply:
Thank you very much for the clear explanation! I got it! 
Indeed, it is good for the user that they can choose from the two ways of quantification by the parameter p.
Thank you so much for the suggestion:-)